frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -242,6 +260,9 @@
           {/if}
 
           <ChangeMemberRoleModal projectId={project.id} bind:this={changeMemberRoleModal} />
+          <AdminContent>
+            <ResetProjectModal bind:this={resetProjectModal} code={data.code} />
```

Comment:
Sure; modals can functionally live anywhere in the DOM tree, after all. I stuck it in between two other modals just because that's where it seemed to make sense, but now that I've merged `develop` into this branch it does make more sense to have just one AdminContent block.